### DISCUSSION TITLE
Add reload/restart operations

### DISCUSSION THREAD

**Discussion by nsleigh** [hassio-addons/addon-vscode#202] [Ideas]
Really like the vscode addon but find myself switching to the default File editor to perform the reload/restart operations. Would it be possible to add this functionality to the vscode addon?

**Comment by frenck** [comment]
It has...

if you open the command pallette and search for "Home Assistant" you'll get tons of things you can restart/reload.

**Reply by nsleigh** [reply]
Of course, didn't think to look there despite using vscode all the time on my desktop!
Thanks
